src/apps/pillarx-app/api/apiSlice.ts

```diff
@@ -5,11 +5,10 @@ export const pillarXApi = createApi({
     reducerPath: 'pillarXApi',
     baseQuery: fetchBaseQuery({ baseUrl: 'https://feed-nubpgwxpiq-uc.a.run.app/' }),
```

Comment:
Just a note for later, in future there will be two sets of API urls, one set for staging and one set for production